README.md

```diff
@@ -76,7 +76,7 @@ In this case these errors are expected and can be safely ignored.
 
```

Comment:
While we're going through I don't think the rest of the Terraform information has been updated against a more recent `terraform-docs` output.

Reply:
You are correct. I decided to skip that step this time around in the interest of time- I intend to update all of our docs when we move to Terraform 1.0 (hopefully shortly after this round of updates ends). At that time, I will also update us to the newest version of terraform-docs.